timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/UseCaseAwareBatchPaxosComponents.java

```diff
@@ -18,24 +18,20 @@
 
 import java.util.EnumMap;
 import javax.ws.rs.NotFoundException;
-import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 
-@Path("/" + PaxosTimeLockConstants.INTERNAL_NAMESPACE)
-public class UseCaseAwareBatchPaxosResource {
+public class UseCaseAwareBatchPaxosComponents {
 
     private final EnumMap<PaxosUseCase, BatchPaxosResources> resourcesByUseCase;
 
-    UseCaseAwareBatchPaxosResource(EnumMap<PaxosUseCase, BatchPaxosResources> resourcesByUseCase) {
+    UseCaseAwareBatchPaxosComponents(EnumMap<PaxosUseCase, BatchPaxosResources> resourcesByUseCase) {
         this.resourcesByUseCase = resourcesByUseCase;
     }
 
-    @Path("{useCase}/" + PaxosTimeLockConstants.BATCH_INTERNAL_NAMESPACE + "/acceptor")
     public BatchPaxosAcceptorResource acceptor(@PathParam("useCase") PaxosUseCase useCase) {
```

Comment:
I assume we don't need the `@PathParam` annotations here anymore, since they're match on the outer resource layer and not here?